sources/packages/web/src/services/EducationProgramOfferingService.ts

```diff
@@ -0,0 +1,22 @@
+import ApiClient from "./http/ApiClient";
+
+export class EducationProgramOfferingService {
+  // Share Instance
+  private static instance: EducationProgramOfferingService;
+
+  public static get shared(): EducationProgramOfferingService {
+    return this.instance || (this.instance = new this());
+  }
+
+  public async createProgramOffering(
+    locationId: number,
+    programId: number,
+    data: any,
```

Comment:
Try to make the type explicit here. My previous PR was missing it due to an issue with Sonar Cloud.

Reply:
Done created a contract